src/components/App.tsx

```diff
@@ -219,25 +242,42 @@ export default function App(): ReactElement {
           <h2>Input Configuration</h2>
           <div className="output-container">
             <div className="row">
-              <Suspense fallback={<div>Loading category select...</div>}>
-                <CategorySelect
-                  id="categories"
-                  onChange={onCategorySelect}
-                  data={state.categories}
-                  defaultValue={getCategoryDefinition(builtin ?? "") ?? null}
-                />
-              </Suspense>
+              <label htmlFor="import-input">
+                <input type="file" id="import-input" onChange={onImport} />
```

Comment:
This seems like a weird hack. Without researching into it, my question is "why can't the button itself trigger the file selector?" If there's a good reason for that question, please leave it as a comment. Otherwise, remove this